src/test/kotlin/com/metalbear/mirrord/utils/IdeaFrame.kt

```diff
@@ -40,13 +41,13 @@ class IdeaFrame(remoteRobot: RemoteRobot, remoteComponent: RemoteComponent) :
         )
 
     val stopDebugging
-        get() = find<ContainerFixture>(
-            byXpath("//div[contains(@myvisibleactions, 'Me')]//div[@myaction.key='action.Stop.text']")
-        )
+        get() = findAll<ContainerFixture>(
+            byXpath("//div[@class='ActionButton' and @myaction='Stop (Stop process)']")
+        ).first()
```

Comment:
why this change?

Reply:
Simple `find` failed for me, I had two of those buttons - one in the toolbar, one in the sidebar of the debugging tab.

Reply:
do you mean locally?

Reply:
wait so it is the same locally and on the CI?

Reply:
I wanted to be prepared for more cases. There was a time that CI failed because IDE for test opened without project tree view expanded. And it has passed before

Reply:
I see sorry for the confusion, now it does open both on the CI and locally?

Reply:
Awesome!